examples/tests/test_examples.py

```diff
@@ -24,20 +24,34 @@
 
 
 # run all tests unless they opt-out
-examples_to_run = find_examples(
-    negative_query="# run_example = false", return_stems=True
-)
+examples_to_run_dict = {
+    path.stem: path for path in find_examples(negative_query="# run_example = false")
+}
+examples_to_run = [x for x in examples_to_run_dict.keys()]
 
 # only test output of examples that opt-in
-examples_to_test = find_examples(query="# test_example = true", return_stems=True)
+examples_to_test_dict = {
+    path.stem: path for path in find_examples(query="# test_example = true")
+}
+examples_to_test = [x for x in examples_to_test_dict.keys()]
 
 
 @pytest.mark.parametrize("module", examples_to_run)
```

Comment:
you can just pass the full Path objects to `@pytest.mark.parametrize` and use `ids` keyword argument like this: `ids=lambda module: module.stem` to simplify